src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Header comments are missing. Be careful not to mention the child classes in the super class header.